Gemfile

```diff
@@ -2,6 +2,7 @@ ruby file: '.ruby-version'
 
 source 'https://rubygems.org'
 
+# rubocop:disable Layout/LineLength
```

Comment:
Mostly unrelated. This allows putting comments after e.g. `gem 'paper_trail',`. Advantage of this discussed below.